app/dashboard/views.py

```diff
@@ -2770,10 +2770,10 @@ def get_profile_tab(request, profile, tab, prev_context):
 
                 return TemplateResponse(request, 'profiles/profile_activities.html', context, status=status)
 
-
-        all_activities = context.get('activities')
+        as_dict = profile.to_dict()
```

Comment:
@gdixon heads up this method takes 3-5s to load in prod, which caused https://github.com/gitcoinco/web/issues/9119

Reply:
use `profile.as_dict` which is already cached/pre-computed in the db

Reply:
https://github.com/gitcoinco/web/pull/9124